ext/src/main/java/io/burt/kafka/clients/KafkaClientsLibrary.java

```diff
@@ -58,6 +59,8 @@ static RubyClass mapErrorClass(Ruby runtime, Throwable t) {
       String javaName = t.getClass().getSimpleName();
       String rubyName = javaName.substring(0, javaName.length() - 9) + "Error";
       return (RubyClass) runtime.getClassFromPath(String.format("Kafka::Clients::%s", rubyName));
+    } else if (t instanceof TimeoutException) {
+      return (RubyClass) runtime.getModule("Timeout").getConstant("Error");
```

Comment:
Especially since this test worked fine locally. So there's something weird going on here.